src/Illuminate/Support/Arr.php

```diff
@@ -657,4 +657,21 @@ public static function wrap($value)
 
         return is_array($value) ? $value : [$value];
     }
+    
+    /**
+     * Loop through multiple arrays in parallel
+     *
+     * @param iterable ...$arrays
+     * @return array
+     */
+    public static function map(callable $callback, ...$arrays)
```

Comment:
This function is actually `zipWith` I think?

Reply:
kind of